### DISCUSSION TITLE
Votación para el uso correcto de siglas y acrónimos en español.

### DISCUSSION THREAD

**Discussion by NGHdeveloper** [reactjs/es.reactjs.org#845] [Polls]
¡Hola comunidad de React!, luego del [debate](https://github.com/reactjs/es.react.dev/discussions/839), generado por el PR #837, solicitamos tengan a bien votar como traducir las siglas en plural del inglés (con letra s o sin ella al final de la sigla, por ejemplo IDs, UIs, etc, versus los ID, las UI, etc. según la RAE.
Desde ya ¡muchas gracias!.

**Comment by carburo** [comment]
Creo que las opciones no están del todo claras o al menos debería existir una tercera. Pienso que en cualquiera de los casos se debería aplicar una regla para mantener la consistencia en la documentación, sea esta o no la misma de la RAE.

**Reply by NGHdeveloper** [reply]
dime cual seria la tercera opción y la agrego a la votación, o como deberían ser las opciones

**Reply by carburo** [reply]
Pluralizar como cualquier otra palabra (con s al final: APIs, IDs, etc) y agregar a la guía de estilo.

**Reply by NGHdeveloper** [reply]
Listo, agregado, ahora mismo anuncio el cambio en  New Translation Progress Checklist #500

**Comment by Byron2016** [comment]
Quiero hacer constar que no hay la referencia en [Glosario](https://github.com/reactjs/es.react.dev/wiki/Glosario) o en [Guia de estilo](https://github.com/reactjs/es.react.dev/wiki/Gu%C3%ADa-de-estilo), entiendo ya se decidió por la pluralización contraviniendo la RAE, en lo personal a mis oídos es mejor pluralizarlo, pero debería constar sea en glosario o en las guías de estilo con su respectivo ejemplo.
